tool/internal/instrument/trampoline.go

```diff
@@ -429,19 +435,22 @@ func (ip *InstrumentPhase) buildTrampolineType(before bool) *dst.FieldList {
 	paramList := &dst.FieldList{List: []*dst.Field{}}
```

Comment:
Could you please add a note to the comment above, clarifying that all grouped parameters (like a, b int) are expanded into separate parameters?
```
    // Build parameter list of trampoline function.
	// For before trampoline, it's signature is:
	// func S(h* HookContext, recv type, arg1 type, arg2 type, ...)
	// For after trampoline, it's signature is:
	// func S(h* HookContext, arg1 type, arg2 type, ...)
    // <comment>
```